app/views/layouts/decidim/_mini_footer.html.erb

```diff
@@ -1,5 +1 @@
-<% if fearlesscities_site? %>
-  <%= render partial: "layouts/fearlesscities/minifooter" %>
-<% else %>
-  <%= render partial: "layouts/decidim/minifooter" %>
-<% end %>
+<%= render partial: "platoniq/mini_footer" %>
```

Comment:
If we remove the partials for the different tennants, how can we add the images of the different organizations to the footer?

Reply:
🤔 Is it the case here? I thought there were no custom images in either of the footers

Reply:
We can change it later if necessary

Reply:
No, but I thought that maybe we can add the logo of bcnencomu. They might ask for it, but ofc we can change it later!